internal/ui/ui_js.go

```diff
@@ -717,6 +718,24 @@ func (u *userInterfaceImpl) Window() Window {
 	return &nullWindow{}
 }
 
+type Monitor struct{}
+
+func (m *Monitor) Bounds() image.Rectangle {
+	return image.Rect(0, 0, window.Get("innerWidth").Int(), window.Get("innerHeight").Int())
```

Comment:
These are not a monitor's size but a frame's size. Let's use [the Screen API](https://developer.mozilla.org/en-US/docs/Web/API/Screen). The values are already in device-independent pixels.

```go
w := windows.Get("screen").Get("width").Int()
h := windows.Get("screen").Get("height").Int() 
return image.Rect(0, 0, w, h)
```